src/test/java/org/apache/commons/lang3/ArrayUtilsTest.java

```diff
@@ -2087,7 +2087,730 @@ public void testReverseObjectRange() {
         assertEquals(null, array);
     }
     
+    //-----------------------------------------------------------------------
```

Comment:
possible simplification for all of your tests is to create a second array ```expected``` and then use ```assertArrayEquals```. IMHO this would be easier to read, because one could see the expected result of the operation directly instead of reconstructing it from the indices.